worker/codalabworker/docker_image_manager.py

```diff
@@ -70,6 +70,8 @@ def touch_image(self, digest):
         logger.debug('Touched image digest=%s at %f', digest, now)
 
     def start_cleanup_thread(self):
+        if self._max_images_bytes is None:
+            return
```

Comment:
How about a quick comment saying that None means infinite room, so no need to cleanup?